en/docs/api-manager/4.5.0/assets/attachments/learn/analytics/geo-location/linux/update-geolocation-data.sh

```
@@ -0,0 +1,53 @@
+        exit 0;
+    elif [ ! -d "$converted_dir" ]; then
+        echo "Cannot find the directory path $converted_dir, execution terminated! "
+        exit 0;
+    elif [ -d "$original_dir" ];then
+        #Get first column form original GeoLite2-City-Blocks-IPv4
+        echo " ** get first column form original"
+        cut -d , -f 1 $original_dir/GeoLite2-City-Blocks-IPv4.csv > first.csv
+
+        #Change column name ‘network’ to ‘network_cidr’
+        echo " ** change column name to ‘network_cidr’"
+        cut -d , -f 1 $original_dir/GeoLite2-City-Blocks-IPv4.csv > first.csv
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)